packages/angular_devkit/schematics/src/tree/host-tree.ts

```diff
@@ -180,7 +180,7 @@ export class HostTree implements Tree {
         case 'c': {
           const { path, content } = action;
 
-          if ((this._willCreate(path) || this._willOverwrite(path))) {
+          if ((this._willCreate(path) || this._willOverwrite(path) || this.exists(path))) {
```

Comment:
This new check determines if the file already exists versus being created or overwritten.